sdks/common/fn-api/src/main/proto/beam_fn_api.proto

```diff
@@ -278,33 +154,20 @@ message ProcessBundleDescriptor {
   // refer to this.
   string id = 1;
 
-  // (Deprecated) A list of primitive transforms that should
-  // be used to construct the bundle processing graph.
-  //
-  // Migrate to Runner API definitions found within transforms field.
-  repeated PrimitiveTransform primitive_transform = 2 [deprecated = true];
-
-  // (Deprecated) The set of all coders referenced in this bundle.
-  //
-  // Migrate to Runner API defintions found within codersyyy field.
-  repeated Coder coders = 4 [deprecated = true];
-
   // (Required) A map from pipeline-scoped id to PTransform.
-  map<string, org.apache.beam.runner_api.v1.PTransform> transforms = 5;
+  map<string, org.apache.beam.runner_api.v1.PTransform> transforms = 2;
```

Comment:
Spoke with @kennknowles, he says MessageWithComponents is the Dataflow specific thing. I am currently discussing with him whether we should take on Components here or not.

In the mean time, this is still forward progression whether we adopt Components here or not. Do you think we can merge now and address migration to Components when that discussion comes to a conclusion.